pages/index.js

```diff
@@ -28,6 +29,6 @@ export async function getStaticProps() {
     props: {
       members: getPublicMembers,
     },
-    revalidate: 432000,
+    revalidate: shared.leaderboard,
```

Comment:
Ini `shared.leaderboard` emang ngereturn undefined ya om?. Dan juga ini kan contributors, jadi ganti sekalian ke contributors.

Saran saya buat const baru di line 27, lalu ganti `shared.leaderboard`.
```js
const EXPIRY_TTL_API = shared.constant.EXPIRY_TTL.contributors; // ganti nama const jika punya nama yang lebih mudah untuk dipahami
```

```suggestion
    revalidate: EXPIRY_TTL_API,
```

Reply:
Oiya seharusnya ke constant nya dulu, sori sori kurang teliti. Kalo leaderboard sama contributors sama om waktunya ku set, sengaja.

Reply:
Hmm ya udah ku set ke contributors aja yah